packages/compiler-cli/ngcc/test/rendering/umd_rendering_formatter_spec.ts

```diff
@@ -20,10 +20,14 @@ import {NgccReferencesRegistry} from '../../src/analysis/ngcc_references_registr
 import {SwitchMarkerAnalyzer} from '../../src/analysis/switch_marker_analyzer';
 import {UmdReflectionHost} from '../../src/host/umd_host';
 import {UmdRenderingFormatter} from '../../src/rendering/umd_rendering_formatter';
+import {AdditionalFormatOptions, createUmdModuleFactory, ParenthesisFormat, WrapperFunctionFormat} from '../helpers/umd_utils';
 import {makeTestEntryPointBundle} from '../helpers/utils';
 
 interface TestFileSpec extends Omit<TestFile, 'contents'> {
-  contents: {preamble?: string; wrapperFunction: string; wrapperCallArguments: string;};
+  contents: {
+    preamble?: string; moduleName: string; dependencies: string[]; factoryBody: string;
+    additionalOptions?: AdditionalFormatOptions,
```

Comment:
```suggestion
    additionalOptions?: AdditionalFormatOptions;
```

(hopefully that helps clang-format in rendering newlines between all properties)

Reply:
😱

Reply:
For the record, no it doesn't help clang-format 🤦‍♂️